tycho-plugins/repository-utils/src/test/java/org/jboss/tools/tycho/sitegenerator/SystemPropertiesTest.java

```diff
@@ -10,14 +10,10 @@
 public class SystemPropertiesTest {
 
 	@Test
-	public void testHostName() {
+	public void testHostName() throws UnknownHostException {
 		java.net.InetAddress localMachine;
-		try {
-			localMachine = java.net.InetAddress.getLocalHost();
-			System.setProperty("HOSTNAME",localMachine.getHostName());
-		} catch (UnknownHostException e) {
-			e.printStackTrace();
-		}
+		localMachine = java.net.InetAddress.getLocalHost();
+		System.setProperty("HOSTNAME",localMachine.getHostName());
```

Comment:
what is this actually testing ?

The test does not seem to test anything besides lookup hostname and store it in a system property.

A system property that will then be set in other tests dependent on what random sequence junit decides to run the tests in ?